src/commcare_cloud/environment/schemas/terraform.py

```diff
@@ -53,6 +53,8 @@ class ServerConfig(jsonobject.JsonObject):
     volume_size = jsonobject.IntegerProperty(default=20)
     block_device = jsonobject.ObjectProperty(lambda: BlockDevice, default=None)
     group = jsonobject.StringProperty()
+    # todo: invert this so that all new machines are bionic unless otherwise specified
```

Comment:
@NitigyaS would it be possible to do this now/soon? What are the machine types that _can't_ yet be provisioned with ubuntu bionic?

Reply:
@sanjay2916 @NitigyaS still curious to know the answer to this

Reply:
I think we should wait for a while unless all machine supports bionic OS. 
So Far only these machine supports bionic
- celery
- webworkers
- pillow
- formplayer
- airflow
- pgbouncer